pctasks/core/pctasks/core/cosmos/container.py

```diff
@@ -279,7 +279,11 @@ def bulk_put(self, models: Iterable[T]) -> None:
                     lambda: self._container_client.scripts.execute_stored_procedure(
                         sp_name,
                         partition_key=partition_key,
-                        params=[list(item_group)],
+                        # azure-cosmos==4.7.0 seems to have an incorrect type hint
```

Comment:
I *think* this is just a bad type hint in the `azure-cosmos` package, but I haven't put together a minimal reproducer yet.

Our unit tests failed when changing thisto `params=item_group`, which would be a `list[dict[str, Any]]`.